apps/civicsignalblog/src/payload/collections/CivicSignalBlog/Authors.js

```diff
@@ -1,7 +1,9 @@
+import canRead from "../../access/applications/researchBlog";
```

Comment:
```suggestion
import canRead from "../../access/applications/research";
```

Similarly for the other apps, make them match what we've defined in `applications.js`